### PR TITLE
dash/math-modifier-demo-tweaks

### PR DESCRIPTION
**Pull request by sebastianbochan**

### CODE REVIEW

File: samples/dashboards/demo/datagrid-mathmodifier/demo.html [hunk 1]
```diff
@@ -7,11 +7,12 @@
...
 <p class="highcharts-description">
-    The dashboard demonstrates some capabilities of the MathModifier. The
-    last column in the table is calculated automatically by a defined
+    The dashboard demonstrates some capabilities of the MathModifier based on data
+    that visualize Euro foreign exchange reference rate to US dollar (USD).
+    The data covers one week of rates in August 2023.
+    More information can be found at the
+    <a href="https://www.ecb.europa.eu/stats/policy_and_exchange_rates/euro_reference_exchange_rates/html/eurofxref-graph-usd.en.html" rel="noreferrer" target="_blank">Euro Data Store</a>
+    of the European Central Bank.
+    Thelast column in the table is calculated automatically by a defined
```
Comment (karolkolodziej) [inline samples/dashboards/demo/datagrid-mathmodifier/demo.html:16]:
```suggestion
    The last column in the table is calculated automatically by a defined
```

**Review by karolkolodziej — APPROVED** [review]
Shouldn't we also change the cypress demo?

**Review by pawellysy — APPROVED** [review]

**Review by Vazonik — APPROVED** [review]

File: samples/dashboards/demo/datagrid-mathmodifier/demo.js [hunk 2]
```diff
@@ -67,15 +68,40 @@ Dashboards.board('container', {
...
+                    zooming: false,
+                    events: {
+                        redraw: function () {
+                            if (!this.series[1].options.yAxis) {
+                                this.series[1].update({
+                                    yAxis: 1
+                                });
+                            }
+                        }
+                    }
```
Comment (bre1470) [inline samples/dashboards/demo/datagrid-mathmodifier/demo.js:80]: What is the purpose of this callback? Is it needed for the demonstration?

Comment (sebastianbochan) [reply samples/dashboards/demo/datagrid-mathmodifier/demo.js:80]: It's the only possible workaround to assign a series to another axis.

**Pull request merged**